src/main/g8/project/Dependencies.scala

```diff
@@ -9,14 +9,14 @@ object Dependencies {
 
     // Scala third-party
     val catsCore   = "1.6.1"
-    val catsEffect = "1.4.1"
-    val circe      = "0.11.1"
+    val catsEffect = "1.4.0"
+    val circe      = "0.13.0"
 
     // Scala first-party
 
     // Testing
-    val specs2     = "4.7.0"
-    val scalaCheck = "1.14.1"
+    val specs2     = "4.7.0" // 4.7.1 bumps cats to 2.0.0
```

Comment:
> 4.7.1 bumps cats to 2.0.0

Not that big deal. Cats 1 and 2 are binary compatible.

Reply:
Done.